### DISCUSSION TITLE
How to add NAS?

### DISCUSSION THREAD

**Discussion by Omnilocus** [rern/rAudio#1040] [Fixed Issues]
How do I add NAS, without getting a "fatal" error? 

Errors: Bad escaped character in JSON at position 1031
-after trying to add NAS.  
And reboot & refresh does not help. Neither is the lack of labels in the GUI ...

Filled in: DS-718\Music and added last digits in adress x...x.**100**
(Also tried:  \\DS-718\Music)

**Comment by cmh714** [comment]
try using forward slashes....thats how I am set up but I use nfs. think cifs is the same.

**Reply by Sagnasty** [reply]
Open Synology File Station.
Drill down to the root directory of your shared music folder.
Right click on this directory and select Properties.
Copy the path listed in "Location" text box.
Paste the copied path as your Share Path when adding your rAudio network storage.   Example: **/volume1/music/music**

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by Omnilocus** [comment]
Thanks, will try

**Comment by Omnilocus** [comment]
Still no go
![Screenshot (1120)](https://user-images.githubusercontent.com/13270561/193100901-4154b86d-2d85-4bd6-8703-8c09b87b5988.png)

**Reply by kw-sudo** [reply]
Hi Omnilocus,

make sure SMB is active on Diskstation

I had the same here. What helped for me:

(r/Audio) I took a closer look at the settings/system/storage...and pushed the question mark button.

![raudiojgkgd](https://user-images.githubusercontent.com/104136223/193307207-d31ac570-2081-42be-aaea-735b2e572bf9.jpg)

Until now I didn't give my NAS a separate name just set the music directory and the user/password. Now I followed the pic's advise, made a directory and adopted the path.

![adonn-nas](https://user-images.githubusercontent.com/104136223/193310065-bfe9fbf7-055d-4fcd-9222-6eba6b8d6b6a.jpg)

Second: fstab...I put in SMB version to vers=3.0 (Synology NAS is capable to min=1.0, max=3.0) e.g.
`//192.168.1.100/music  /mnt/MPD/NAS/YOURNASNAME  cifs  noauto,username=xxxxx,password=xxxxxxx,uid=45,gid=45,iocharset=utf8  0  0`

Two reboots and directory is remount. Up to now it works.

**Reply by Omnilocus** [reply]
Thanks

**Reply by ceejayemm** [reply]
@rern 
 A missing name for the CIFS/NFS storage dialogue box has caused numerous issues for a while.  Is it time to make it Mandatory and avoid such issues ?

Chris
